pkgs/cupertino_http/lib/src/cupertino_web_socket.dart

```diff
@@ -106,6 +106,10 @@ class CupertinoWebSocket implements WebSocket {
   /// Handle an incoming message from the peer and schedule receiving the next
   /// message.
   void _handleMessage(URLSessionWebSocketMessage value) {
+    if (_events.isClosed) {
+      return;
+    }
```

Comment:
[optional] I like to omit the braces for guard clauses when they fit one one line.

```suggestion
    if (_events.isClosed) return;
```